drivers/sx127x/sx127x.c

```diff
@@ -307,9 +307,10 @@ static void _on_tx_timeout(void *arg)
 
 static void _on_rx_timeout(void *arg)
 {
-    netdev_t *dev = (netdev_t *) arg;
+    (void) arg;
+    //netdev_t *dev = (netdev_t *) arg;
 
-    dev->event_callback(dev, NETDEV_EVENT_RX_TIMEOUT);
+    //dev->event_callback(dev, NETDEV_EVENT_RX_TIMEOUT);
```

Comment:
TODO? Please mark the PR as WIP if so.

Reply:
This is actually a bug in upstream. The callback here is called from an xtimer timeout rather than the DIO1 interrupt. I will open a PR with the proper fix.

Reply:
Then I propose to fix the bug in upstream first instead of adding commented out code (and wrong comment style for that matter ;-))